modules/update-duration-example.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/understanding-openshift-update-duration.adoc
+
+:_content-type: CONCEPT
+[id="update-duration-example_{context}"]
+= Example update duration of cluster Operators
+
+image::update-duration.png[A diagram displaying the periods during which cluster Operators update themselves during an OCP update]
```

Comment:
Maybe put the image after the introductory paragraph so that folks know what the image is?

Reply:
I could see how that would give some context to the image, but it would also make users read about specific references to parts of the image in these following paragraphs without having the chance to see the image itself.

I think the section heading is at least enough for users to know what they're looking at before getting to the textual description, and would want to keep the image where it is, but let me know if you feel strongly about the image placement.